Project/Source/GameplaySystems.cpp

```diff
@@ -207,8 +207,23 @@ float Screen::GetScreenHeight() {
 	return static_cast<float>(App->window->GetHeight());
 }
 
+void Screen::SetResolution(int width_, int height_) {
+	App->window->SetSize(width_, height_);
```

Comment:
Does Window->SetSize automatically triggers the viewportResized?

Reply:
Yes